[EDU-6432] Edge AI reference (#1624)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/edge-ai/ai-models/mistral-3-small.mdx

---
title: Mistral 3 Small (24B AWQ)
description: >-
  Mistral 3 Small provides a range of capabilities, including text generation, image analysis, embeddings, and more.
meta_tags: 'edge ai, ai models, artificial intelligence, edge computing, mistral'
namespace: docs_edge_ai_models_mistral_3_small
permalink: /documentation/products/ai/edge-ai/models/mistral-3-small/
---

> This is a language model that delivers capabilities comparable to larger models while being compact. It is ideal for conversational agents, function calling, fine-tuning, and local inference with sensitive data.

Reviewer: Normalmente tu começa pelo nome do modelo nos acima...Mistral 3 Small is a language model...?
```suggestion
This is a language model that delivers capabilities comparable to larger models while being compact. It's ideal for conversational agents, function calling, fine-tuning, and local inference with sensitive data.
```

> This is a language model that delivers capabilities comparable to larger models while being compact. It is ideal for conversational agents, function calling, fine-tuning, and local inference with sensitive data.

Author: ```suggestion
**Mistral 3 Small** is a language model that delivers capabilities comparable to larger models while being compact. It is ideal for conversational agents, function calling, fine-tuning, and local inference with sensitive data.
```

## Model details

| Category | Details |
|----------|---------|
| **Model Name** | Mistral 3 Small |
| **Version** | 24B AWQ |
| **Model Category** | Large Language Model (LLM) |
| **Size** | 24B parameters |
| **HuggingFace Model** | [casperhansen/mistral-small-24b-instruct-2501-awq](https://huggingface.co/casperhansen/mistral-small-24b-instruct-2501-awq) |
| **OpenAI Compatible Endpoint** | [Chat Completions](https://platform.openai.com/docs/api-reference/chat/create) |

> | **LICENSE** | [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/) |

Reviewer: Pq License tá em maiúsculo?

> | **LICENSE** | [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/) |

Author: ```suggestion
| **License** | [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/) |
```

                            "type": "string",
                            "description": "The city and state"
                        }
                    },
                    "required": ["location"]
                }
            }
        }
    ]
}'
```

### Multimodal (text + image) example

This is an example of a multimodal request using this model:

```bash
curl http://endpoint-url/v1/chat/completions \
 -H "Content-Type: application/json" \
 -d '{
    "stream": true,
    "model": "casperhansen/mistral-small-24b-instruct-2501-awq",
    "max_tokens": 1024,
    "messages": [
        {"role": "system", "content": "You are a helpful assistant."},
        {"role": "user", "content": [
            {"type": "text", "text": "What is in this image?"},
            {"type": "image_url", "image_url": {"url": "https://example.com/image.jpg"}}
        ]}
    ]
}'
```

## Performance

The model has been quantized using AWQ, resulting in a 50% reduction in model size while maintaining performance. It supports a large context window of 32,000 tokens, making it suitable for processing long documents and maintaining context in conversations.

### Running with Edge Functions:

> This is a code example of how to use this model through Edge Functions:

Reviewer: Acima foi usado "This is an example of how to run this model with Edge Functions:". Padronizar?

> This is a code example of how to use this model through Edge Functions:

Author: ```suggestion
This is an example of how to use this model with Edge Functions:
```
